tests/driver/driver_test.py

```diff
@@ -677,6 +699,55 @@ def test_execute_schedule_in_throughput_mode(self, es):
             self.assertEqual("docs", sample.total_ops_unit)
             self.assertEqual(1, sample.request_meta_data["bulk-size"])
 
+    @mock.patch("elasticsearch.Elasticsearch")
+    def test_execute_schedule_with_progress_determined_by_runner(self, es):
+        es.bulk.return_value = {
+            "errors": False
+        }
+
+        params.register_param_source_for_name("driver-test-param-source", DriverTestParamSource)
+        test_track = track.Track(name="unittest", description="unittest track",
+                                 indices=None,
+                                 challenges=None)
+
+        task = track.Task("time-based", track.Operation("time-based", operation_type="unit-test-recovery", params={
+            "indices-to-restore": "*",
+            # The runner will determine progress
+            "size": None
+        },
+                                                        param_source="driver-test-param-source"),
```

Comment:
nit: indentation seems a bit wonky here

Reply:
Thanks! Addressed in https://github.com/elastic/rally/pull/789/commits/b1c5b724e8f15c85272d727a59de156c551c18f2.